sql: TestDropWhileBackfill failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/70e3468e7ed5fb30b10eaaf972acbb0f16099d01

Parameters:
```
TAGS=
GOFLAGS=-race
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestDropWhileBackfill PKG=github.com/cockroachdb/cockroach/pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1179814&tab=buildLog

```
I190315 08:03:14.291456 96604 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190315 08:03:14.292327 96693 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:39967: context canceled
I190315 08:03:14.293814 96694 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:39319: context canceled
I190315 08:03:14.295997 96604 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190315 08:03:14.297050 96606 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190315 08:03:14.297692 96605 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] intent_resolver_ir_batcher
1      [async] intent_resolver_gc_batcher
1      [async] closedts-rangefeed-subscriber
I190315 08:03:14.298499 96605 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] intent_resolver_gc_batcher
1      [async] closedts-rangefeed-subscriber
I190315 08:03:14.298877 96605 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190315 08:03:14.302959 96604 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190315 08:03:14.307454 96607 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
W190315 08:03:14.312472 96010 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = context canceled:
I190315 08:03:14.326053 96699 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:35317: context canceled
W190315 08:03:14.326433 95853 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = context canceled:
I190315 08:03:14.327046 96608 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190315 08:03:14.327721 96698 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36945: context canceled
I190315 08:03:14.327916 96608 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190315 08:03:14.328901 96700 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:43721: context canceled
W190315 08:03:14.329799 95878 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
I190315 08:03:14.330167 96701 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:39967: context canceled
I190315 08:03:14.331071 96607 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190315 08:03:14.331553 96607 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190315 08:03:14.331781 96605 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190315 08:03:14.388260 96706 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:39319: context canceled
I190315 08:03:14.388477 96708 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:35317: context canceled
I190315 08:03:14.388860 96707 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:43721: context canceled
I190315 08:03:14.389565 96709 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36945: context canceled
--- FAIL: TestDropWhileBackfill (15.74s)
    schema_changer_test.go:833: pq: internal error: uncaught error: context deadline exceeded

```

SHA: https://github.com/cockroachdb/cockroach/commits/04ef15974085e14f758b20c552a84052eac9fa2b

Parameters:
```
TAGS=
GOFLAGS=
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestDropWhileBackfill PKG=github.com/cockroachdb/cockroach/pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1180876&tab=buildLog

```
4      [async] closedts-subscription
1      [async] intent_resolver_ir_batcher
1      [async] closedts-rangefeed-subscriber
I190316 05:38:44.200161 98217 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
W190316 05:38:44.200548 97633 storage/raft_transport.go:583  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = context canceled:
W190316 05:38:44.200629 96333 gossip/gossip.go:1496  [n3] no incoming or outgoing connections
W190316 05:38:44.201141 97649 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = context canceled:
W190316 05:38:44.201254 97564 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W190316 05:38:44.201785 96621 gossip/gossip.go:1496  [n4] no incoming or outgoing connections
W190316 05:38:44.201849 97531 storage/raft_transport.go:583  [n4] while processing outgoing Raft queue to node 1: EOF:
W190316 05:38:44.201941 97542 storage/raft_transport.go:583  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
I190316 05:38:44.202003 98215 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
I190316 05:38:44.202115 98218 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] intent_resolver_gc_batcher
1      [async] closedts-rangefeed-subscriber
I190316 05:38:44.202145 98216 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
I190316 05:38:44.202445 98218 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 05:38:44.202529 98216 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190316 05:38:44.202540 98219 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
I190316 05:38:44.202593 98217 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 05:38:44.202676 98218 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
I190316 05:38:44.202694 98216 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190316 05:38:44.202705 98217 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190316 05:38:44.202780 98216 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190316 05:38:44.202785 98218 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190316 05:38:44.202806 98218 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190316 05:38:44.202826 98218 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190316 05:38:44.202950 98217 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190316 05:38:44.203032 98217 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190316 05:38:44.205951 98230 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37207: context canceled
I190316 05:38:44.206200 98195 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33889: context canceled
I190316 05:38:44.206298 98196 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:39971: context canceled
I190316 05:38:44.206410 98229 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34619: context canceled
--- FAIL: TestDropWhileBackfill (5.42s)
    schema_changer_test.go:833: pq: internal error: uncaught error: context deadline exceeded

```

SHA: https://github.com/cockroachdb/cockroach/commits/04ef15974085e14f758b20c552a84052eac9fa2b

Parameters:
```
TAGS=
GOFLAGS=-race
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestDropWhileBackfill PKG=github.com/cockroachdb/cockroach/pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1181174&tab=buildLog

```
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.052661 96636 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
W190316 07:11:47.055737 95973 storage/raft_transport.go:583  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I190316 07:11:47.060183 96635 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] intent_resolver_gc_batcher
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.060558 96635 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.061850 96633 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      ts.poller: poll
I190316 07:11:47.062524 96633 util/stop/stopper.go:546  quiescing; tasks left:
1      ts.poller: poll
1      [async] closedts-subscription
I190316 07:11:47.062948 96633 util/stop/stopper.go:546  quiescing; tasks left:
1      ts.poller: poll
I190316 07:11:47.063935 96635 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
I190316 07:11:47.064173 96636 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.064440 96635 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190316 07:11:47.064791 96636 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.064927 96635 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190316 07:11:47.065555 96636 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190316 07:11:47.100929 96623 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:35937: context canceled
I190316 07:11:47.101110 96624 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:44673: context canceled
I190316 07:11:47.101414 96625 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42643: context canceled
I190316 07:11:47.101716 96622 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36033: context canceled
W190316 07:11:47.113283 94650 ts/db.go:211  [n2,ts-poll] error writing time series data: node unavailable; try another peer
I190316 07:11:47.134067 96695 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36033: context canceled
I190316 07:11:47.134344 96697 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42643: context canceled
I190316 07:11:47.134535 96694 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:44673: context canceled
I190316 07:11:47.135622 96696 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:45435: context canceled
I190316 07:11:47.155843 96626 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42643: context canceled
I190316 07:11:47.156155 96627 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36033: context canceled
I190316 07:11:47.156579 96741 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:45435: context canceled
I190316 07:11:47.163060 96628 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:35937: context canceled
--- FAIL: TestDropWhileBackfill (12.61s)
    schema_changer_test.go:833: pq: internal error: uncaught error: context deadline exceeded

```

SHA: https://github.com/cockroachdb/cockroach/commits/c32bec552dd89861fbc899a662c1c0cb78acc921

Parameters:
```
TAGS=
GOFLAGS=-race
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestDropWhileBackfill PKG=github.com/cockroachdb/cockroach/pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1182276&tab=buildLog

```
1      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190317 06:45:46.675510 96704 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190317 06:45:46.675864 96703 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-rangefeed-subscriber
I190317 06:45:46.676287 96704 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190317 06:45:46.677658 96705 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190317 06:45:46.679083 96705 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
W190317 06:45:46.715436 94704 server/node.go:851  [n2,summaries] error recording status summaries: node unavailable; try another peer
I190317 06:45:46.727134 96706 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:34377: context canceled
I190317 06:45:46.729273 96707 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:36237: context canceled
I190317 06:45:46.729765 96708 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:38795: context canceled
I190317 06:45:46.730133 96709 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39429: context canceled
I190317 06:45:46.738953 96655 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:44067: context canceled
I190317 06:45:46.739230 96657 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:36237: context canceled
I190317 06:45:46.739624 96658 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:34377: context canceled
I190317 06:45:46.739851 96656 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39429: context canceled
I190317 06:45:46.768255 96659 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:44067: context canceled
I190317 06:45:46.768748 96660 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39429: context canceled
I190317 06:45:46.769983 96758 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:38795: context canceled
I190317 06:45:46.770751 96757 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:36237: context canceled
I190317 06:45:46.774661 96674 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:38795: context canceled
I190317 06:45:46.775869 96671 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:44067: context canceled
I190317 06:45:46.776895 96673 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:36237: context canceled
I190317 06:45:46.778123 96672 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:34377: context canceled
I190317 06:45:46.804307 96776 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:34377: context canceled
I190317 06:45:46.804777 96775 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:38795: context canceled
I190317 06:45:46.805760 96777 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39429: context canceled
I190317 06:45:46.822763 94439 kv/transport_race.go:113  transport race promotion: ran 31 iterations on up to 1191 requests
--- FAIL: TestDropWhileBackfill (12.02s)
    schema_changer_test.go:833: pq: internal error: uncaught error: context deadline exceeded

```

SHA: https://github.com/cockroachdb/cockroach/commits/c2939ec9a4f15b7fb8683a5805deeb241953e7aa

Parameters:
```
TAGS=
GOFLAGS=-race
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestDropWhileBackfill PKG=github.com/cockroachdb/cockroach/pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1183412&tab=buildLog

```
I190318 07:03:29.636982 96700 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] transport racer
I190318 07:03:29.637585 96700 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      [async] transport racer
I190318 07:03:29.640990 96704 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      ts.poller: poll
1      [async] intent_resolver_gc_batcher
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.641489 96704 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      ts.poller: poll
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.641757 96703 util/stop/stopper.go:546  quiescing; tasks left:
4      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.641004 96702 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190318 07:03:29.642529 96702 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] closedts-subscription
I190318 07:03:29.643772 96700 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] transport racer
1      [async] closedts-subscription
I190318 07:03:29.644796 96700 util/stop/stopper.go:546  quiescing; tasks left:
1      [async] transport racer
I190318 07:03:29.645440 96704 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      ts.poller: poll
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.645723 96703 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.646061 96704 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      ts.poller: poll
1      [async] closedts-rangefeed-subscriber
I190318 07:03:29.646615 96703 util/stop/stopper.go:546  quiescing; tasks left:
3      [async] closedts-subscription
I190318 07:03:29.647006 96703 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
I190318 07:03:29.646637 96704 util/stop/stopper.go:546  quiescing; tasks left:
2      [async] closedts-subscription
1      ts.poller: poll
I190318 07:03:29.647429 96704 util/stop/stopper.go:546  quiescing; tasks left:
1      ts.poller: poll
1      [async] closedts-subscription
I190318 07:03:29.647808 96704 util/stop/stopper.go:546  quiescing; tasks left:
1      ts.poller: poll
I190318 07:03:29.672192 96640 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:36727: context canceled
I190318 07:03:29.672227 96641 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:43465: context canceled
I190318 07:03:29.672575 96642 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:34885: context canceled
I190318 07:03:29.672820 96643 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:32907: context canceled
I190318 07:03:29.681317 94082 kv/transport_race.go:113  transport race promotion: ran 65 iterations on up to 1266 requests
W190318 07:03:29.693183 95501 ts/db.go:211  [n5,ts-poll] error writing time series data: node unavailable; try another peer
I190318 07:03:29.747818 96676 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:43465: context canceled
I190318 07:03:29.749047 96726 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:36727: context canceled
I190318 07:03:29.750192 96727 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:38265: context canceled
I190318 07:03:29.750260 96677 rpc/nodedialer/nodedialer.go:143  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:34885: context canceled
--- FAIL: TestDropWhileBackfill (15.31s)
    schema_changer_test.go:833: pq: internal error: uncaught error: context deadline exceeded

```

--- a/pkg/sql/schema_changer.go
+++ b/pkg/sql/schema_changer.go
@@ -192,10 +192,9 @@ func isPermanentSchemaChangeError(err error) bool {
 			return false
 
 		case pgerror.CodeInternalError:
-			if err.Message == context.DeadlineExceeded.Error() {
+			if strings.Contains(err.Message, context.DeadlineExceeded.Error()) {
 				return false
 			}
-
 		}
 	}
 

